### PR TITLE
fix: Don't display int64 range in docs

### PR DESCRIPTION
**Pull request by SB-harshitajadhav**
## Motivation and Context

[STOP-231](https://smartbear.atlassian.net/browse/STOP-231)

## Description

Previously range was getting displayed without being defined in the schema for int64.

## How Has This Been Tested?

Tested locally by doing yalc in Platform internal.

## Screenshot(s)/recordings(s)

Before
image

After 
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] This PR's code follows as closely as possible [the coding style/guidelines of this project](???).
- [ ] I have added error reporting and followed [the error reporting guidelines](???).
- [ ] I have added event tracking and followed [the event tracking guidelines](???).
- [ ] I have updated any relevant documentation accordingly to reflect this PR's changes.
- [ ] I have added automated tests (unit/integration/e2e/other) to cover my changes.
- [x] All new and existing tests pass locally (excluding flaky CI tests).

[STOP-231]: https://smartbear.atlassian.net/browse/STOP-231?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/shared/__tests__/Validations.spec.tsx [hunk 1]
```diff
@@ -127,5 +127,17 @@ describe('Validations component', () => {
 
       expect(wrapper).toHaveTextContent('>= 0');
     });
+
```
Comment (prafullaAtSB) [inline src/components/shared/__tests__/Validations.spec.tsx:130]: as a part of the walkthrough .. the test case needs to be recorrected to identify the location of the min value and max value.  This test case is passing without the fix as well.

Comment (SB-harshitajadhav) [reply src/components/shared/__tests__/Validations.spec.tsx]: Hi Prafulla, I have removed the testcase as this issue is not reproducible in json-schema-viewer and for platform internal there will be a playwright testcase.

**Review by bhaskarsontakke — APPROVED** [review]
As mentioned by Prafulla, please provide the detail why we have removed int64 min/max calculation. changes looks ok to me

**Review by SB-pradeep-bande — APPROVED** [review]
Looks good to me.

**Pull request merged**